src/Engines/Extensions/engine.multiRender.ts

```diff
@@ -73,7 +82,7 @@ ThinEngine.prototype.unBindMultiColorAttachmentFramebuffer = function(textures:
         gl.drawBuffers(attachments);
     }
 
-    for (var i = 0; i < textures.length; i++) {
+    for (var i = 0; i < textures[0]._attachments!.length; i++) {

```

Comment:
[NIT] extract `const count = textures[0]._attachments!.length;` at the top so you could use count here as well.